packages/vite/src/node/importGlob.ts

```diff
@@ -261,3 +257,36 @@ function error(pos: number) {
   err.pos = pos
   throw err
 }
+
+function glob(pattern: string, cwd: string): string[] {
```

Comment:
better to call `cwd` as `base` here? So we avoid possible confusion with `cwd` not being `cwd` when modified.